server/pypi/packages/scipy/meta.yaml

```diff
@@ -1,6 +1,12 @@
+{% if PY_VER == "3.8" %}
+    {% set numpy_version = "1.19.5" %}
+{% elif PY_VER in ["3.9", "3.10", "3.11"] %}
+    {% set numpy_version = "1.23.3" %}
+{% endif %}
```

Comment:
There's no need for this, because we aren't building new packages for Python 3.8 anymore.

Reply:
Addressed!